railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_2.rb.tt

```diff
@@ -14,6 +14,11 @@
 # Existing cookies will be converted on read then written with the new scheme.
 # Rails.application.config.action_dispatch.use_authenticated_cookie_encryption = true
 
+# Embeds the experity of a cookie inside its value.
+# This makes the cookie value incompatible with Rails < 5.2 so set to false if your
+# cookie needs to be read by a Rails < 5.2 application.
+# Rails.application.config.action_dispatch.embed_expirity_in_the_cookie_value = true
```

Comment:
I don't think expirity is a real word 😬 

Maybe `use_embedded_cookie_expiration`?

Reply:
I'd go with `embed_expiry_in_signed_and_encrypted_cookies`. Or `use_authenticated_encryption_and_embed_expiry_for_cookies` if we're unifying with `use_authenticated_cookie_encryption`.